core/src/main/kotlin/di/Module.kt

```diff
@@ -70,6 +70,7 @@ import org.eclipse.apoapsis.ortserver.services.OrganizationService
 import org.eclipse.apoapsis.ortserver.services.OrtRunService
```

Comment:
Commit message nit: Start title with capital letter, i.e. "Implement project service..."
Also, there seem to be leading whitespace in some rows.